bower.json

```diff
@@ -38,7 +38,8 @@
     "angular-ui-select": "0.17.1",
     "JsBarcode": "jsbarcode#3.5.9",
     "angular-touch": "1.6.5",
-    "angular-growl-notifications": "2.5.0"
+    "angular-growl-notifications": "2.5.0",
+    "webcam-directive": "*"
```

Comment:
It is best practice to put an exact version number, if possible.  That way, we can choose to upgrade the directive when we want to.  Otherwise, if they introduce breaking changes, Travis CI will break randomly on someone's PR.